packages/math-input/src/components/key-handlers/key-translator.ts

```diff
@@ -82,6 +106,10 @@ const keyToMathquillMap: Record<Key, MathQuillUpdaterCallback> = {
         mathQuill.keystroke("Left"); // under the root
     },
 
+    FRAC: (mathQuill) => {
+        mathQuill.cmd("\\frac");
+    },
```

Comment:
Can this just use `buildGenericCallback("\\frac", ActionType.CMD)`?

Reply:
Good catch!